lib/github-auth.js

```diff
@@ -2,14 +2,15 @@ var querystring = require('querystring');
 var request = require('request');
 var autosave = require('json-autosave');
 var serverSecrets;
+var baseUrl = process.env.BASE_URL || "https://shields.io";
```

Comment:
However, this is used by the Shields server to redirect `http://img.shields.io` to `http://shields.io`. I guess the use case here is that you want to redirect the base of your image server to your homepage. In that case a distinction might be desired?

Perhaps we can make the distinction a bit more clear by renaming them to `HOME_PAGE` and `IMAGE_BASE_URL` or sth.